app/src/main/java/org/p2p/wallet/splash/SplashPresenter.kt

```diff
@@ -19,7 +22,19 @@ class SplashPresenter(
             sellInteractor.loadSellAvailability()
             delay(MINIMUM_SPLASH_SHOWING_TIME_MS)
 
-            openRootScreen()
+            loadPricesAndBids()
+        }
+    }
+
+    private fun loadPricesAndBids() {
+        launch {
+            try {
+                userInteractor.loadAllTokensData()
+            } catch (e: Throwable) {
+                Timber.e(e, "Error loading initial tokens data")
```

Comment:
maybe show error screen on error? to reload data.... i mean check if we have something in data base and if nothing and we have an error do not pass user in?

Reply:
We didn't show it before.
Right now, I am not sure if it's okat to show error on Splash screen.
He will see an empty on the main. 
Anyway let's talk separately on it